src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/BedSearchTest.kt

```diff
@@ -228,7 +228,7 @@ class BedSearchTest : IntegrationTestBase() {
                 resultsPremisesCount = 1,
                 resultsBedCount = 1,
                 results = listOf(
-                  createTemporaryAccommodationBedSearchResult(premises, room, bed, searchPdu.name, 1, listOf(), listOf()),
+                  createTemporaryAccommodationBedSearchResult(premises, room, bed, searchPdu.name, 1, 0, listOf(), listOf()),
```

Comment:
i appreciate it's probably a pain to fix but there's a lot of nondescript arguments here (and in other tests) that i think would be helped by specifying argument names for clarity? something to consider going forward maybe

Reply:
I agree I will change it by adding the names of arguments